toolkit/tools/internal/userutils/userutils.go

```diff
@@ -77,6 +89,46 @@ func AddUser(username string, hashedPassword string, uid string, installChroot *
 	return nil
 }
 
+func UpdateUserPassword(installRoot, username, hashedPassword string) error {
```

Comment:
Is there a unit test in customizer that covers this? Not sure what the best way to test is without making the security scanners angry though.

Reply:
Added some UTs.